spec/java_buildpack/jre/ibmjre_like_spec.rb

```diff
@@ -0,0 +1,59 @@
+# Encoding: utf-8
+
+# Cloud Foundry Java Buildpack
+# Copyright 2017 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'spec_helper'
+require 'component_helper'
+require 'java_buildpack/component/mutable_java_home'
+require 'java_buildpack/jre/ibmjre_like'
+
+describe JavaBuildpack::Jre::IbmjreLike do
```

Comment:
Please follow the formatting style of the other tests.  This means whitespace between specs, etc.